test/gurka/test_match.cc

```diff
@@ -25,23 +22,62 @@ TEST(Standalone, BasicMatch) {
   const auto layout = gurka::detail::map_to_coordinates(ascii_map, gridsize);
   auto map = gurka::buildtiles(layout, ways, {}, {}, "test/data/basic_match");
 
-  auto result = gurka::match(map, {"1", "2", "3", "4", "5"}, false, "auto");
+  auto result = gurka::match(map, {"1", "2", "3", "4", "5"}, "via", "auto");
 
   gurka::assert::osrm::expect_match(result, {"AB", "BC", "CD"});
-  gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg_Maneuver_Type_kStart,
-                                                DirectionsLeg_Maneuver_Type_kContinue,
-                                                DirectionsLeg_Maneuver_Type_kRight,
-                                                DirectionsLeg_Maneuver_Type_kDestination});
-
-  // the shape is correct!?
-  float len = 0;
-  for (const auto& route : result.trip().routes()) {
-    for (const auto& leg : route.legs()) {
-      auto points = midgard::decode<std::vector<midgard::PointLL>>(leg.shape());
-      len += midgard::length(points);
-    }
-  }
-  EXPECT_NEAR(len, 100.f, 1.f);
+  gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg::Maneuver::kStart,
+                                                DirectionsLeg::Maneuver::kContinue,
+                                                DirectionsLeg::Maneuver::kRight,
+                                                DirectionsLeg::Maneuver::kDestination});
 
   gurka::assert::raw::expect_path_length(result, 0.100, 0.001);
+}
+
+TEST(Standalone, UturnMatch) {
```

Comment:
@kevinkreiser did ya'll identify any utrecht integration test cases to use as well or do you need help identifying one of those?

Reply:
this change is entirely data independent so it didnt occur to me that we needed an utrecht test. it would be very easy to craft some if you want to though, so please be my guest!

Reply:
Here's an example that I believe is in the boundary of the test tiles. I'll try it out locally and push a commit to this branch: 

```json
{"shape":[{"lat":"52.0908896","lon":"5.0959038","type":"break","radius":15,"search_cutoff":21000000,"preferred_side":"either"},
{"lat":"52.0911440","lon":"5.0985641","type":"break","radius":15,"search_cutoff":21000000,"preferred_side":"either"},
{"lat":"52.0909154","lon":"5.0959570","type":"break","radius":15,"search_cutoff":21000000,"preferred_side":"either"},
{"lat":"52.0911629","lon":"5.0984494","type":"break","radius":15,"search_cutoff":21000000,"preferred_side":"either"}],
"costing":"auto","shape_match":"map_snap","format":"osrm","shape_format":"geojson","filters":{},"trace_options":{"interpolation_distance":0},"force_traffic":false}
```

Points (lon,lat): 
```
5.0959038,52.0908896
5.0985641,52.0911440
5.0959570,52.0909154
5.0984494,52.0911629
```

![Screen Shot 2020-05-22 at 2 52 25 PM](https://user-images.githubusercontent.com/353255/82712026-ef4c9b00-9c3b-11ea-99bd-e5ac224bbaf4.png)
![Screen Shot 2020-05-22 at 2 52 38 PM](https://user-images.githubusercontent.com/353255/82712029-f1aef500-9c3b-11ea-8697-434333248ad0.png)